common/src/main/java/com/wynntils/services/itemfilter/statproviders/GearTypeStatProvider.java

```diff
@@ -16,4 +18,23 @@ public List<String> getValue(WynnItem wynnItem) {
 
         return List.of(gearTypeItemProperty.getGearType().name());
     }
+
+    @Override
+    public List<String> getValidInputs() {
+        // Can't use all values of the enum as WEAPON, ACCESSORY, MASTERY_TOME and CHARM
+        // don't work with this filter, ItemTypeStatProvider should be used for tomes and charms
```

Comment:
`ItemTypeStatProvider` doesn't work for WEAPON, ACCESSORY?

Reply:
No, you can only do `itemType:gear`, there is no `WeaponItem` or `AccessoryItem`

Reply:
Maybe we should implement that. Probably not here though.